app/views/passengers/brochure.html.haml

```diff
@@ -1,76 +1,75 @@
-%h1 SpecTrans Passenger Information
-.row
-  .col-md-6
-    %table.table
-      %caption Contact Information
-      %thead
-      %tbody
-        %tr
-          %th Phone
-          %td 413-545-2086
-        %tr
-          %th Public Website
-          %td= link_to 'www.umass.edu/transportation/special-transportation',
-            'https://www.umass.edu/transportation/special-transportation'
-        %tr
-          %th Email
-          %td stspv@umass.edu
-.row
-  .col-md-6
-    %table.table
-      %caption Special Transportation Staff
-      %thead
-      %tbody
-        %tr
-          %th Supervisors
-          %td Lisa Gagnon &amp; Shayla Keane
-        %tr
-          %th Coordinator
-          %td Diana Noble
-%h2 Hours of Operation
-.row
-  .col-md-6
-    %table.table
-      %caption Semester
-      %thead
-        %tr
-          %th{ 'aria-label': 'Days' }
-          %th Dispatch
-          %th Van
-      %tbody
-        %tr
-          %th{ scope: 'row'} Monday Through Friday
-          %td 7:00 AM to 5:00 PM
-          %td 7:15 AM to 8:15 PM
-        %tr
-          %th{ scope: 'row' } Saturday
-          %td none
-          %td 10:00 AM to 8:00 PM
-        %tr
-          %th{ scope: 'row' } Sunday
-          %td none
-          %td 10:00 AM to 8:00 PM
-.row
-  .col-md-6
-    %table.table
-      %caption Reduced Service (Holidays, Summer, Intersession)
-      %thead
-        %tr
-          %th{ 'aria-label': 'Days' }
-          %th Dispatch
-          %th Van
-      %tbody
-        %tr
-          %th{ scope: 'row'} Monday Through Friday
-          %td 8:00 AM to 5:00 PM
-          %td 7:15 AM to 7:15 PM
-        %tr
-          %th{ scope: 'row' } Saturday
-          %td none
-          %td 11:15 AM to 4:45 PM
-        %tr
-          %th{ scope: 'row' } Sunday
-          %td closed
-          %td closed
-= render 'policies'
-= render 'scheduling_rides_information'
+#brochure.container
+  %h1 SpecTrans Passenger Information
+  .row
+    .col-lg-6
+      %table.table
+        %caption Contact Information
+        %thead
+        %tbody
+          %tr
+            %th Phone
+            %td 413-545-2086
+          %tr
+            %th Public Website
+            %td= link_to 'www.umass.edu/transportation/special-transportation',
+              'https://www.umass.edu/transportation/special-transportation'
+          %tr
+            %th Email
+            %td stspv@umass.edu
+    .col-lg-6
+      %table.table
+        %caption Special Transportation Staff
+        %thead
+        %tbody
+          %tr
+            %th Supervisors
+            %td Lisa Gagnon &amp; Shayla Keane
```

Comment:
Not a layout change, but these are not the current supervisors anymore. Should be Aurora O'Connor (maybe Rory, I know they tend to prefer that name), Karina Patterson (outgoing) and Zack Suranofsky (incoming).

Reply:
Yes, I have a different PR pending with _content_ updates. Primarily ST -> AVS, but I also intend to ask Aurora to once-over the brochure.

Reply:
I didn't see the PR in my notifs, I'll look through the repo.